sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/util/GcsUtil.java

```diff
@@ -467,15 +467,17 @@ SeekableByteChannel open(GcsPath path, GoogleCloudStorageReadOptions readOptions
    * @param type the type of object, eg "text/plain".
    * @return a Callable object that encloses the operation.
    */
-  public WritableByteChannel create(GcsPath path, String type) throws IOException {
-    return create(path, type, uploadBufferSizeBytes);
+  public WritableByteChannel create(GcsPath path, String type, boolean expectFileToNotExist)
```

Comment:
Don't merge, I think I need to change this so there is public api stability.
I'll add a new flavor with an options class so we don't end up with more down the line.

Reply:
Done, PTAL @chamikaramj